crates/pixi_progress/src/lib.rs

```diff
@@ -45,6 +45,7 @@ pub fn default_progress_style() -> indicatif::ProgressStyle {
 
 /// Returns the style to use for a progressbar that is indeterminate and simply shows a spinner.
 pub fn long_running_progress_style() -> indicatif::ProgressStyle {
+    #[allow(clippy::literal_string_with_formatting_args)]
```

Comment:
probably because of the rust toolchain bump; is this a false positive or an actual problem?